tests/unit/test_target_snowflake.py

```diff
@@ -85,4 +85,86 @@ def test_persist_40_records_with_batch_wait_limit(self, dbSync_mock, flush_strea
         target_snowflake.persist_lines(self.config, lines)
 
         # Expecting flush after every records + 1 at the end
-        assert flush_streams_mock.call_count == 41
\ No newline at end of file
+        assert flush_streams_mock.call_count == 41
+
+    @patch('target_snowflake.DbSync')
+    @patch('target_snowflake.flush_streams')
+    def test_archive_load_files(self, flush_streams_mock, dbSync_mock):
+        self.config['id'] = 'test-tap-id'
+        self.config['archive_load_files.enabled'] = True
+        self.config['db_conn.s3_bucket'] = 'dummy_bucket'
```

Comment:
```suggestion
        self.config['s3_bucket'] = 'dummy_bucket'
```

Separating by dots is not valid and here we don't use the original YAML file. The config is a generated JSON of the items under `db_conn` and some other propertes listed [here](https://github.com/transferwise/pipelinewise/blob/eca52d4fa566f27c53b206a59ada86ee7d986a68/pipelinewise/cli/config.py#L320)